icu4c/source/common/unicode/ucnv.h

```diff
@@ -344,8 +344,6 @@ ucnv_compareNames(const char *name1, const char *name2);
  * other than its an alias starting with the letters "cp". Please do not
  * associate any meaning to these aliases.</p>
  *
- * \snippet samples/ucnv/convsamp.cpp ucnv_open
- *
```

Comment:
if the sample worked, would it be helpful to have a _compiling_ sample here in the API docs? That was the purpose of it